help/include/how-to-start-a-new-topic.md

```diff
@@ -58,3 +58,9 @@
    button in the bottom right corner of the app.
 
 {end_tabs}
+
+!!! tip ""
```

Comment:
Let's move this tip inside the Web/Desktop instructions tab, so that we don't have to explain where it applies.